smoke-test/src/main/java/org/opennms/smoketest/containers/GrafanaContainer.java

```diff
@@ -0,0 +1,28 @@
+package org.opennms.smoketest.containers;
+
+
+import org.junit.Before;
+import org.opennms.smoketest.utils.TestContainerUtils;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.wait.strategy.WaitStrategy;
+import org.testcontainers.lifecycle.TestLifecycleAware;
+import org.testcontainers.utility.DockerImageName;
+
+public class GrafanaContainer extends GenericContainer<GrafanaContainer> implements TestLifecycleAware {
+
+    public static final int WEB_PORT = 3000;
+    public static final String GRAFANA_ALIAS = "grafana";
+    public GrafanaContainer() {
+        super("grafana/grafana:latest");
```

Comment:
can't use latest for tests. Fix a version.

Reply:
Which version we should use.

Reply:
Version 11.3.0+security-01 (commit: 5ddc329279bb66e3fd584812b8a4e16751eeecd0, branch: HEAD)

Reply:
Added and tested with 11.3.0 .